Solution for number verification issue in current version of Whatsapp Web

vphelipe opened the pull request:
# PR Details

Solution for number verification issue in current version of Whatsapp Web

UPDATED: 23/11/2022

Comment 
## Description

Updated QueryExist module and how to use it.

## Related Issue

Old Query Exists module stopped working in the current version of whatsapp web, it is not possible to check if numbers are registered in whatsapp.

vphelipe committed:
Update Injected.js

vphelipe committed:
Update Client.js

Update getNumberId() function

vphelipe committed:
Update Injected.js

Updated QueryExist module

ariona commented:
This still return null on registered number

vphelipe commented:
> This still return null on registered number

Call me on whatsapp 556182615929

ariona commented:
I Also tried this, but all number, even invalid number return something instead of null
```js
var isValid= window.mR.findModule('queryExist')[0].queryExist;
var wid = sebaranstore.WidFactory.createWid('628111111111@c.us')
await isValid('628111111111',wid)
```
will return
```json
{
    "wid": {
        "server": "c.us",
        "user": "628111111111",
        "_serialized": "628111111111@c.us"
    },
    "biz": false,
    "disappearingMode": {
        "duration": 0,
        "settingTimestamp": 0
    }
}
```

vphelipe commented:
> I Also tried this, but all number, even invalid number return something instead of null
> 
> ```js
> var isValid= window.mR.findModule('queryExist')[0].queryExist;
> var wid = sebaranstore.WidFactory.createWid('628111111111@c.us')
> await isValid('628111111111',wid)
> ```
> 
> will return
> 
> ```json
> {
>     "wid": {
>         "server": "c.us",
>         "user": "628111111111",
>         "_serialized": "628111111111@c.us"
>     },
>     "biz": false,
>     "disappearingMode": {
>         "duration": 0,
>         "settingTimestamp": 0
>     }
> }
> ```

Change to:

var isValid= window.mR.findModule('queryExist')[0].queryExist;
var wid = sebaranstore.WidFactory.createWid('628111111111@c.us')
await isValid('phone',wid)

ariona commented:
I think adding + sign solve the issue, there also no need to create wid anymore
```js
var isValid= window.mR.findModule('queryExist')[0].queryExist;
await isValid('phone',"+628123456789")
```

vphelipe commented:
> yeah, also tried that first, it return `null` for valid & invalid number

Change to:

const result = await window.Store.QueryExist('phone',"+5599999999999");

Putting + in front of the number indicates that the number has an international code, removing the + will verify the number by automatically inserting your country code.

ariona commented:
> > yeah, also tried that first, it return `null` for valid & invalid number
> 
> Change to:
> 
> const result = await window.Store.QueryExist('phone',"+5599999999999");
> 
> Putting + in front of the number indicates that the number has an international code, removing the + will verify the number by automatically inserting your country code.

i see, that make sense. I can confirm it working :)

vphelipe committed:
Update QueryExist parameters

Putting + in front of the number indicates that the number has an international code, removing the + will verify the number by automatically inserting your country code.

ViktorEmma commented:
Than you [vphelipe](https://github.com/vphelipe) 

But there are many devices did not get the new version yet because it is still beta.
Most devices currently are 2.2243.7
So the pull must handle different versions.
Thus, it is perfect.

burkass commented:
I got below error

Error: Evaluation failed: TypeError: e.isLid is not a function
    at t.queryExists [as QueryExist] (https://web.whatsapp.com/bootstrap_qr.f232d10a9d2fea971276.js:50:91256)
    at __puppeteer_evaluation_script__:4:40

herduin commented on src/Client.js:
```diff
@@ -1042,8 +1042,8 @@ class Client extends EventEmitter {
-            const wid = window.Store.WidFactory.createWid(number);
-            const result = await window.Store.QueryExist(wid);
+            //const wid = window.Store.WidFactory.createWid(number);
+            const result = await window.Store.QueryExist("phone","+"number);
```
Sintax error... 
```
const result = await window.Store.QueryExist( "phone" , "+" + number );
```
But with this works for me...

XxQuickSilverZz replied:
**This works for me...**

vphelipe committed:
Update getNumberId()

herduin reviewed (approved):
Tested and working...

ariona reviewed (approved):
Tested and working, invalid number return `null`, valid number return object with `wid`

tlgsn commented:
Tested and working...
Thx.

tuyuribr commented on src/Client.js:
```diff
@@ -1037,13 +1037,13 @@ class Client extends EventEmitter {
-        if (!number.endsWith('@c.us')) {
-            number += '@c.us';
-        }
+        //if (!number.endsWith('@c.us')) {
```
remove comments them we are good to commit

vphelipe replied:
Ok

tuyuribr commented on src/Client.js:
```diff
@@ -1037,13 +1037,13 @@ class Client extends EventEmitter {
-            const wid = window.Store.WidFactory.createWid(number);
-            const result = await window.Store.QueryExist(wid);
+            //const wid = window.Store.WidFactory.createWid(number);
+            const result = await window.Store.QueryExist('phone',`+${number}`);
```
Maybe check if it already starts with "+" if not appent "+"

vphelipe replied:
Ok

tuyuribr commented on src/util/Injected.js:
```diff
@@ -26,7 +26,7 @@ exports.ExposeStore = (moduleRaidStr) => {
     window.Store.MessageInfo = window.mR.findModule('sendQueryMsgInfo')[0];
     window.Store.OpaqueData = window.mR.findModule(module => module.default && module.default.createFromData)[0].default;
-    window.Store.QueryExist = window.mR.findModule('queryExists')[0].queryExists;
+    window.Store.QueryExist = window.mR.findModule('queryExist')[0].queryExist; //new module
```
remove comment

vphelipe replied:
ok

tuyuribr commented:
Tested here, it's working

vphelipe committed:
Update Client.js

vphelipe committed:
Update Injected.js

elasti-co commented:
Please approve PR. Many people are waiting for this fix. 🙏🏼

wictornogueira commented:
For compatibility sake, add a little check to see if `number` ends with `@c.us` and remove it if it does:
```js
if (number.endsWith('@c.us'))
  number = number.slice(0, -5);
```

agrinfauzi commented:
good job sir, thankyou 🥰

ariona commented:
@vphelipe  Not working again for latest update **2.2245.9** 😦, i haven't got the update though

soulideth commented:
now. Working ?

XxQuickSilverZz commented:
> now. Working ?

not for me...

jestergods commented:
@vphelipe would you please give me your phone number or write to me, I get the no number error.
my phone number is: +1 (651) 427-7618 @vphelipe

VictorArgote reviewed (approved):
Tested these changes and worked for me.

XxQuickSilverZz commented:
Please tell me all the changes. I try it and it not work. Which version are you using? (Libary and WhatsApp).

Can you give me a example number from you it returns you yes?
Am 22. Nov. 2022, 21:03 +0100 schrieb Victor Argote ***@***.***>:
> @VictorArgote approved this pull request.
> Tested these changes and worked for me.
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: ***@***.***>

vphelipe committed:
Update QueryExist Parameters

New way to send parameters to the QueryExist function

vphelipe commented:
I sent an update with the new way to send parameters to the QueryExist function.

vphelipe commented:
> @vphelipe Not working again for latest update **2.2245.9** 😦, i haven't got the update though

The solution has been implemented

laar789 reviewed (approved):
It works for me. I'm using this changes currently in four numbers

Shvan11 commented:
Working for me :)

onurguven commented:
Tested and working.

Lucia361 commented:
> > @vphelipe Not working again for latest update **2.2245.9** 😦, i haven't got the update though
> 
> The solution has been implemented

It does not work if you use WA Business Account to verify the Numbers.

You will get 

`Error: user must have an id or a phone at t.USyncUser.validate (https://web.whatsapp.com/app.77ee8aa7a4c593101105.js:50`

XxQuickSilverZz commented:
I get this error with a normal number not business…
SAME ERROR!!

Its not working!
Am 23. Nov. 2022, 17:36 +0100 schrieb Lucia Fuskova ***@***.***>:
> > > @vphelipe Not working again for latest update 2.2245.9 😦, i haven't got the update though
> > The solution has been implemented
> It does not work if you use WA Business Account to verify the Numbers.
> You will get
> Error: user must have an id or a phone at t.USyncUser.validate (https://web.whatsapp.com/app.77ee8aa7a4c593101105.js:50
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: ***@***.***>

vphelipe commented:
> I get this error with a normal number not business… SAME ERROR!! Its not working! Am 23. Nov. 2022, 17:36 +0100 schrieb Lucia Fuskova ***@***.***>:
> […](#)
> > > @vphelipe Not working again for latest update 2.2245.9 😦, i haven't got the update though > The solution has been implemented It does not work if you use WA Business Account to verify the Numbers. You will get Error: user must have an id or a phone at t.USyncUser.validate (https://web.whatsapp.com/app.77ee8aa7a4c593101105.js:50 — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

Check the latest update

jtourisNS commented on src/Client.js:
```diff
@@ -1037,13 +1037,9 @@ class Client extends EventEmitter {
-            const wid = window.Store.WidFactory.createWid(number);
-            const result = await window.Store.QueryExist(wid);
+            if (number.endsWith('@c.us')) number.replace('@c.us','');
+            const result = await window.Store.QueryExist({type: 'phone', phone: number.startsWith('+') ? number : `+${number}`});
```
This line is incorrect. Instead of sending an object as a param with properties `type` and `phone`, WhatsApp internals are expecting 2 parameters here to work as expected instead an object. Changing this line to the following should fix the problem:

const result = await window.Store.QueryExist('phone', number.startsWith('+') ? number : `+${number}`);

Lucia361 replied:
Still Not Working. 

`user must have an id or a phone
    at t.USyncUser.validate (app.77ee8aa7a4c593101105.js:50:58574)
    at app.77ee8aa7a4c593101105.js:50:50786
    at Array.filter (<anonymous>)
    at t.USyncQuery._buildStanza (app.77ee8aa7a4c593101105.js:50:50718)
    at app.77ee8aa7a4c593101105.js:50:51431
    at Generator.next (<anonymous>)
    at t (vendor1~app.58a51d93a250f83ab555.js:2:66483)
    at s (vendor1~app.58a51d93a250f83ab555.js:2:66694)
    at vendor1~app.58a51d93a250f83ab555.js:2:66753
    at Y (app.77ee8aa7a4c593101105.js:40:119783)`

jtourisNS replied:
<img width="899" alt="Screen Shot 2022-11-23 at 2 49 59 PM" src="https://user-images.githubusercontent.com/39593002/203615204-ac3b167b-4699-444d-9db4-de75593e424c.png">

Yes, it works. Make sure you are not using a cached version of npm modules

vphelipe replied:
> This line is incorrect. Instead of sending an object as a param with properties `type` and `phone`, WhatsApp internals are expecting 2 parameters here to work as expected instead an object. Changing this line to the following should fix the problem:
> 
> ```
> const result = await window.Store.QueryExist('phone', number.startsWith('+') ? number : `+${number}`);
> ```

Depending on your region, the queryExist function receives parameters in different ways. An update has been made to deal with this.

jtouris reviewed (changes requested):
Thanks for the PR 

However, this fix is still not working with latest versions of WhatsApp. Please take a look at the following change, I tested it and worked as expected.

vphelipe committed:
Update Client.js

Update for compatibility with all versions of the queryExist function

ViktorEmma commented:
Hi , thank all 
My friend works for the WhatsApp team in the technical support service, and he told me that they have a bug in this versions 2.2244.5 beta and the current final version is 2.2244.6.


So the solution method is **2.2243.7 + 2.2245.9**
Like the old method found in the library.
**2.2244.5 + 2.2244.6** like the way he mentioned [vphelipe](https://github.com/vphelipe)

The final solution is to wait 24-48 hours for a final new release > 2.2245.9
The solution method will be the same as the old one and will work for everyone.


Note: wa.me/number works with the current beta version **2.2245.9** and will also be available in the final version.

Good luck everyone and thanks for your efforts.

ViktorEmma commented:
Hi , thank all
My friend works for the WhatsApp team in the technical support service, and he told me that they have a bug in this versions 2.2244.5 beta and the current final version is 2.2244.6.

So the solution method for 2.2243.7 + 2.2245.9
Same as the current method in the library in [v1.18.3](https://github.com/pedroslopez/whatsapp-web.js/releases/tag/v1.18.3).
2.2244.5 + 2.2244.6 like the way he mentioned [vphelipe](https://github.com/vphelipe)

The final solution is to wait 24-48 hours for a final [new release ](https://web.whatsapp.com/check-update?version=2.2244.6&platform=web)> 2.2245.9
The solution method will be the same as the old one and will work for everyone.

Note: wa.me/number works with the current beta version 2.2245.9 and will also be available in the final version.

Good luck everyone and thanks for your efforts

Shinhwe commented on src/Client.js:
```diff
@@ -1038,7 +1038,8 @@ class Client extends EventEmitter {
     async getNumberId(number) {
         return await this.pupPage.evaluate(async number => {
-            const result = await window.Store.QueryExist('phone', number.startsWith('+') ? number : `+${number}`);
+            if (number.endsWith('@c.us')) number.replace('@c.us','');
```
should be number = number.replace('@c.us','');

rusdiyuqing commented:
not working for me, can help me.. 

thanks.

ViktorEmma commented:
For now   "[currentVersion](currentVersion)": "2.2245.9"

ViktorEmma commented:
Now [currentVersion](https://web.whatsapp.com/check-update?version=2.2245.9&platform=web) is "2.2245.9"

tlgsn commented:
Not working now :/

Shvan11 commented:
Not working now ):

tlgsn commented:
Client.js 1039;

`async getNumberId(number) {
        if (!number.endsWith('@c.us')) {
            number += '@c.us';
        }

        return await this.pupPage.evaluate(async number => {
            const wid = window.Store.WidFactory.createWid(number);
            const result = await window.Store.QueryExist(wid);
            if (!result || result.wid === undefined) return null;
            return result.wid;
        }, number);
    }`

Injected.js 29

`window.Store.QueryExist = window.mR.findModule('queryExists')[0].queryExists;`

working.

Shvan11 commented:
> Client.js 1039;
> 
> async getNumberId(number) { if (!number.endsWith('@c.us')) { number += '@c.us'; }
> 
> ```
>     return await this.pupPage.evaluate(async number => {
>         const wid = window.Store.WidFactory.createWid(number);
>         const result = await window.Store.QueryExist(wid);
>         if (!result || result.wid === undefined) return null;
>         return result.wid;
>     }, number);
> ```
> 
> }
> 
> Injected.js 29
> 
> `window.Store.QueryExist = window.mR.findModule('queryExists')[0].queryExists;`
> 
> working.

not working

jestergods commented:
![error](https://user-images.githubusercontent.com/118081249/203981161-ddfeeacf-e59d-4715-89cf-ac783a6cf378.png)
After reading the qr, I get this error, please help. @vphelipe @tuyuribr @tlgsn @pedroslopez @vphelipe

EDSL1312 commented:
Not working

Someone can help?

Skiroll5 commented:
not working.

XxQuickSilverZz commented:
IT WORKS FOR ME.

@everyone. 
1. Clear the NPM Folder.
2. npm cache clean --force
3. npm install 


my prepare number function:

```
function transformPhoneNumber(phonenumber) {
    if (!phonenumber || phonenumber.indexOf('@c.us') > -1) {
        return phonenumber;
    }
    if (!phonenumber.startsWith('49')) {
        phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', '');
    }

    phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, '');

    phonenumber = phonenumber + '@c.us';
    return phonenumber; // alle special characters weg
}
```

Its for Germany (+49)

Then phonenumber ? await client.isRegisteredUser(phonenumber) : false.

it works fine.

Skiroll5 commented:
> 

I've just tried this getNumberId(countryCode + phoneNumber + "@c.us") and it is returning null.
Any Ideas?

Skiroll5 commented:
> IT WORKS FOR ME.
> 
> @everyone.
> 
> 1. Clear the NPM Folder.
> 2. npm cache clean --force
> 3. npm install
> 
> my prepare number function:
> 
> ```
> function transformPhoneNumber(phonenumber) {
>     if (!phonenumber || phonenumber.indexOf('@c.us') > -1) {
>         return phonenumber;
>     }
>     if (!phonenumber.startsWith('49')) {
>         phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', '');
>     }
> 
>     phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, '');
> 
>     phonenumber = phonenumber + '@c.us';
>     return phonenumber; // remove special chars
> }
> ```
> 
> Its for Germany (+49)
> 
> Then` phonenumber ? await client.isRegisteredUser(phonenumber) : false.`
> 
> it works fine.

I've just tried this `client.getNumberId(countryCode + phoneNumber + "@c.us")` and it is returning null.
Any Ideas?

XxQuickSilverZz commented:
> > IT WORKS FOR ME.
> > @everyone.
> > 
> > 1. Clear the NPM Folder.
> > 2. npm cache clean --force
> > 3. npm install
> > 
> > my prepare number function:
> > ```
> > function transformPhoneNumber(phonenumber) {
> >     if (!phonenumber || phonenumber.indexOf('@c.us') > -1) {
> >         return phonenumber;
> >     }
> >     if (!phonenumber.startsWith('49')) {
> >         phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', '');
> >     }
> > 
> >     phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, '');
> > 
> >     phonenumber = phonenumber + '@c.us';
> >     return phonenumber; // remove special chars
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Its for Germany (+49)
> > Then` phonenumber ? await client.isRegisteredUser(phonenumber) : false.`
> > it works fine.
> 
> I've just tried this `client.getNumberId(countryCode + phoneNumber + "@c.us")` and it is returning null. Any Ideas?

lastest whatsapp version + wweb.js version?

jestergods commented:
> BENİM İÇİN ÇALIŞIYOR.
> 
> @everyone.
> 
> 1. NPM Klasörünü temizleyin.
> 2. npm önbellek temizleme --force
> 3. npm kurulum
> 
> hazırlama numaram işlevi:
> 
> ```
> function transformPhoneNumber(phonenumber) {
>     if (!phonenumber || phonenumber.indexOf('@c.us') > -1) {
>         return phonenumber;
>     }
>     if (!phonenumber.startsWith('49')) {
>         phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', '');
>     }
> 
>     phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, '');
> 
>     phonenumber = phonenumber + '@c.us';
>     return phonenumber; // remove special chars
> }
> ```
> 
> Almanya için (+49)
> 
> O zamanlar` phonenumber ? await client.isRegisteredUser(phonenumber) : false.`
> 
> iyi çalışıyor.

exactly which file I'm going to write this to in which stir

XxQuickSilverZz commented:
> > BENİM İÇİN ÇALIŞIYOR.
> > @everyone.
> > 
> > 1. NPM Klasörünü temizleyin.
> > 2. npm önbellek temizleme --force
> > 3. npm kurulum
> > 
> > hazırlama numaram işlevi:
> > ```
> > function transformPhoneNumber(phonenumber) {
> >     if (!phonenumber || phonenumber.indexOf('@c.us') > -1) {
> >         return phonenumber;
> >     }
> >     if (!phonenumber.startsWith('49')) {
> >         phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', '');
> >     }
> > 
> >     phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, '');
> > 
> >     phonenumber = phonenumber + '@c.us';
> >     return phonenumber; // remove special chars
> > }
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Almanya için (+49)
> > O zamanlar` phonenumber ? await client.isRegisteredUser(phonenumber) : false.`
> > iyi çalışıyor.
> 
> exactly which file I'm going to write this to in which stir

No this is in your code to manipulate the phonenumber before you call isRegisteredUser(...)

Skiroll5 commented:
> > > IT WORKS FOR ME.
> > > @everyone.
> > > 
> > > 1. Clear the NPM Folder.
> > > 2. npm cache clean --force
> > > 3. npm install
> > > 
> > > my prepare number function:
> > > ```
> > > function transformPhoneNumber(phonenumber) {
> > >     if (!phonenumber || phonenumber.indexOf('@c.us') > -1) {
> > >         return phonenumber;
> > >     }
> > >     if (!phonenumber.startsWith('49')) {
> > >         phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', '');
> > >     }
> > > 
> > >     phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, '');
> > > 
> > >     phonenumber = phonenumber + '@c.us';
> > >     return phonenumber; // remove special chars
> > > }
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > Its for Germany (+49)
> > > Then` phonenumber ? await client.isRegisteredUser(phonenumber) : false.`
> > > it works fine.
> > 
> > I've just tried this `client.getNumberId(countryCode + phoneNumber + "@c.us")` and it is returning null. Any Ideas?
> 
> lastest whatsapp version + wweb.js version? 

Yes, but I've edited the files as said before, does that matter?

XxQuickSilverZz commented:
Sure. Delete npm folder and install again everything.
Am 25. Nov. 2022, 23:40 +0100 schrieb Skiroll5 ***@***.***>:
> > > > IT WORKS FOR ME.
> > > > @everyone.
> > > >
> > > > 1. Clear the NPM Folder.
> > > > 2. npm cache clean --force
> > > > 3. npm install
> > > >
> > > > my prepare number function:
> > > > function transformPhoneNumber(phonenumber) {
> > > >    if (!phonenumber || ***@***.***') > -1) {
> > > >        return phonenumber;
> > > >    }
> > > >    if (!phonenumber.startsWith('49')) {
> > > >        phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', '');
> > > >    }
> > > >
> > > >    phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, '');
> > > >
> > > >    phonenumber = phonenumber + ***@***.***';
> > > >    return phonenumber; // remove special chars
> > > > }
> > > > Its for Germany (+49)
> > > > Then phonenumber ? await client.isRegisteredUser(phonenumber) : false.
> > > > it works fine.
> > > I've just tried this client.getNumberId(countryCode + phoneNumber + ***@***.***") and it is returning null. Any Ideas?
> > lastest whatsapp version + wweb.js version?
> Yes, but I've edited the files as said before, does that matter?
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you commented.Message ID: ***@***.***>

Skiroll5 commented:
> Sure. Delete npm folder and install again everything. Am 25. Nov. 2022, 23:40 +0100 schrieb Skiroll5 ***@***.***>:
> […](#)
> > > > IT WORKS FOR ME. > > > @everyone. > > > > > > 1. Clear the NPM Folder. > > > 2. npm cache clean --force > > > 3. npm install > > > > > > my prepare number function: > > > function transformPhoneNumber(phonenumber) { > > > if (!phonenumber || ***@***.***') > -1) { > > > return phonenumber; > > > } > > > if (!phonenumber.startsWith('49')) { > > > phonenumber = phonenumber.replace(/^.{1}/g, '49').replace('+', ''); > > > } > > > > > > phonenumber = phonenumber.trim().replace(/[^a-zA-Z0-9]/g, ''); > > > > > > phonenumber = phonenumber + ***@***.***'; > > > return phonenumber; // remove special chars > > > } > > > Its for Germany (+49) > > > Then phonenumber ? await client.isRegisteredUser(phonenumber) : false. > > > it works fine. > > I've just tried this client.getNumberId(countryCode + phoneNumber + ***@***.***") and it is returning null. Any Ideas? > lastest whatsapp version + wweb.js version? Yes, but I've edited the files as said before, does that matter? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you commented.Message ID: ***@***.***>

Thank you very much, it works now.

satriax commented:
Hi All
I've updated to 1.18.4 and the problem is gone. 
Cheers

purpshell committed:
Merge branch 'main' into main

PurpShell commented:
@vphelipe @tuyuribr how is the status of this PR?

Fix the ESLint error so I can do an approving review

tuyuribr commented:
> @vphelipe @tuyuribr how is the status of this PR?
> 
> Fix the ESLint error so I can do an approving review

It seems that was an expontaneous bug in waweb that they fixed some iterations ago

PurpShell commented:
> > @vphelipe @tuyuribr how is the status of this PR?
> > Fix the ESLint error so I can do an approving review
> 
> It seems that was an expontaneous bug in waweb that they fixed some iterations ago

Then it is OK if I close. I will re-open otherwise
